consensus/polybft/consensus_runtime.go

```diff
@@ -259,19 +263,20 @@ func (c *consensusRuntime) populateFsmIfBridgeEnabled(
 }
 
 // FSM creates a new instance of fsm
-func (c *consensusRuntime) FSM() (*fsm, error) {
+func (c *consensusRuntime) FSM() error {
 	// figure out the parent. At this point this peer has done its best to sync up
 	// to the head of their remote peers.
-	parent, epoch := c.getLastBultBlockAndEpoch()
-
-	if !epoch.Validators.ContainsNodeID(c.config.Key.NodeID()) {
```

Comment:
Don't we need this check?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/865/commits/1d887e7738aa86a285d3ad490808352f95389448